usaspending_api/disaster/v2/views/agency/spending.py

```diff
@@ -57,16 +56,33 @@ def route_agency_spending_backend(request, *args, **kwargs):
     return route_agency_spending_backend
 
 
-class SpendingByAgencyViewSet(PaginationMixin, SpendingMixin, FabaOutlayMixin, DisasterBase):
+class SpendingByAgencyViewSet(
+    PaginationMixin,
+    SpendingMixin,
+    FabaOutlayMixin,
+    ElasticsearchAccountDisasterBase,
+    ElasticsearchSpendingPaginationMixin,
```

Comment:
Can `PaginationMixin` and `DisasterBase` be removed?

Reply:
yeah